.gitmodules

```
@@ -734,3 +734,6 @@
 [submodule "plugins/discourse-category-experts"]
 	path = plugins/discourse-category-experts
 	url = https://github.com/discourse/discourse-category-experts
+[submodule "plugins/discourse-brightcove"]
+	path = plugins/discourse-brightcove
+	url = https://github.com/discourse/discourse-brightcove
```

One tip I learned recently:

```suggestion
	url = https://github.com/discourse/discourse-brightcove
	branch = main
```

---

What benefit does this provide? I assume if it's unspecified, it will just use the default branch? (which is `main`)

---

I believe that when unspecified it uses detached head:

```
# ~/discourse/all-the-plugins/plugins/discourse-zoom
❯ git status
HEAD detached at ed84fed
nothing to commit, working tree clean
```